developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst

```diff
@@ -28,12 +28,29 @@ See `Github issue <https://github.com/nextcloud/server/issues/32060>`_.
 Front-end changes
 -----------------
 
-tbd
+Disabled keyboard shortcuts
```

Comment:
This change can be mentioned here but should find a place in the regular documentation. Maybe in the front-end parts or a11y section if we have one.

Reply:
Already did that back then: https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/js.html#keyboard-shortcuts

Reply:
Then I suggest to create a ref and link to them so one can jump into the details

Reply:
![grafik](https://user-images.githubusercontent.com/213943/200557695-3e7351a2-c89c-4ba3-bf28-4a5ebacc3478.png)


Done